.github/workflows/checkup-kubevirt-vm-latency.check.yaml

```
@@ -90,7 +90,7 @@ jobs:
         working-directory: ./checkups/kubevirt-vm-latency
         run: ./automation/make.sh --build-checkup --build-checkup-image
       - name: Start cluster
-        run: ./automation/make.sh --e2e -- --install-kind --install-kubectl --create-cluster
+        run: ./automation/make.sh --e2e -- --install-kind --install-kubectl --create-multi-node-cluster;
```

Do we still need the `--create-cluster` option (and its logic)?

nit:
```suggestion
        run: ./automation/make.sh --e2e -- --install-kind --install-kubectl --create-multi-node-cluster
```

---

There may be cases where a single node cluster is better fit in terms of CI time, such the echo checkup.

---

Thanks for the answer.